src/renderer/viz/expressions/transformation/Rotation.js

```diff
@@ -0,0 +1,73 @@
+import BaseExpression from '../base';
+import { checkType, implicitCast, checkMaxArguments } from '../utils';
+
+/**
+ * Rotate. Define a rotation in degrees.
+ *
+ * Limitation: only supported in combination with `symbol:`.
+ *
+ * @param {Number} angle - angle to rotate in degrees in clockwise direction
+ * @return {Transform}
+ *
+ * @example <caption>Rotate 30 degrees in clockwise direction.</caption>
+ * const s = carto.expressions;
+ * const viz = new carto.Viz({
+ *   symbol: cross
+ *   transform: s.rotate(30)
+ * });
+ *
+ * @example <caption>Rotate 30 degrees in clockwise direction. (String)</caption>
```

Comment:
Maybe, I would add an example with negative input only to explain that it will be counter-clockwise, but this is just a suggestion.

Reply:
:+1: